packages/blocks/src/page-block/default/selection-manager.ts

```diff
@@ -235,4 +234,16 @@ export class DefaultSelectionManager {
     this._signals.updateFrameSelectionRect.dispose();
     this._mouseDisposeCallback();
   }
+  blockSelected(selectionRect: DOMRect) {
+    this.state.type = 'block';
+    this.state.refreshRichTextBoundsCache(this._container);
+    const { richTextCache } = this.state;
+    const selectedRichTexts = filterSelectedRichText(
+      richTextCache,
+      selectionRect
+    );
+    this.state.selectedRichTexts = selectedRichTexts;
+    const selectedBounds: DOMRect[] = [selectionRect];
+    this._signals.updateSelectedRects.emit(selectedBounds);
```

Comment:
@doodlewind This selection is a bit different from the existing ones, the `selectedBounds` include not only RiechText. I think it looks better now.